CHANGELOG.md

```diff
@@ -23,7 +23,8 @@
 * Publish `ERROR` events from `<RouteErrorBoundary>` so handler modules can react to them. Refs STCOR-455.
 * Dispatch `setOkapiReady` when all resources are loaded. Fixes STCOR-506.
 * Configure `swr`. Refs STCOR-516.
-* Callouts are opened after page reload. Refs STCOR-518. 
+* Callouts are opened after page reload. Refs STCOR-518.
+* Use `==` where possible for more efficient queries.
```

Comment:
It seems that it's forgotten to add `Refs STCOR-520` here.

Reply:
Thanks for the reminder. I discovered this while working on something else, so I just fixed it first and then wrote the ticket after. Shhhh! Don't tell my scrum-master!!!